yt_dlp/extractor/youtube/_video.py

```diff
@@ -3527,6 +3527,8 @@ def gvs_pot_required(policy, is_premium_subscriber, has_player_token):
                 'language_preference': PREFERRED_LANG_VALUE if is_original else 5 if is_default else -10 if is_descriptive else -1,
                 # Strictly de-prioritize damaged and 3gp formats
                 'preference': -10 if is_damaged else -2 if itag == '17' else None,
+                'initRange': fmt.get('initRange'),
+                'indexRange': fmt.get('indexRange'),
             }
```

Comment:
These are not valid format fields; see the `InfoExtractor` docstring for a list of all of them.

We do add new format fields sometimes, but these fields aren't even going to be used by yt-dlp code, so I see no reason to do so

Reply:
Also, returning arbitrary format fields like this will cause any applicable download tests to fail

Reply:
> We do add new format fields sometimes, but these fields aren't even going to be used by yt-dlp code, so I see no reason to do so

but these will be useful to embedders (like me) who want to add initialization segments to their custom dash playlists.


> Also, returning arbitrary format fields like this will cause any applicable download tests to fail

well the youtube download test fails early because of #12263 (ran with `hatch test Youtube)`